### ISSUE TITLE
Check impact of additionalBinding-Extension in R5+

### ISSUE THREAD

**Issue by alexzautke** [FirelyTeam/firely-net-sdk#2251]
See https://build.fhir.org/ig/FHIR/fhir-tools-ig/ValueSet-additional-binding-purpose.html
It should be investigated how it impacts our validation.

**Comment by ewoutkramer** [comment]
Yes, I made a mental note of asking Grahame. I cannot really understand what the extension is doing, so I'll just ask him how it influences validation.

**Comment by ewoutkramer** [comment]
Ok, so the relevant bits for us to look at are:

* Purpose `maximum` - comparable to http://build.fhir.org/extension-elementdefinition-maxvalueset.html, basically this additional binding extension is replacing the use of the maxvalueset extension, since it is a superset.
* Purpose `conformance` - switch to another binding given a validation context.

Most interestingly, this introduces the concept of a "validation context" in terms of a `UsageContext` (http://hl7.org/fhir/R4/metadatatypes.html#UsageContext). Which means the validator needs this context as an input parameter (and an extra argument to the $validate operation).

**Comment by alexzautke** [comment]
Note that in R5 the extension has been moved to native elements of ElementDefinition: https://build.fhir.org/elementdefinition.html
